src/components/NewMovie/NewMovie.tsx

```diff
@@ -48,6 +125,7 @@ export const NewMovie = () => {
             type="submit"
             data-cy="submit-button"
             className="button is-link"
+            disabled={isEmpty === false}
```

Comment:
Try to rewrite this way 👇 
```suggestion
            disabled={!isEmpty}
```

or this way(check the logic). In this case no need to compare with boolean.
```suggestion
            disabled={isEmpty}
```